src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -885,4 +887,36 @@ default IWebhook getWebhookByID(String id) {
 	 * @return The count of members on the guild
 	 */
 	int getTotalMemberCount();
+
+	/**
+	 * Gets the full audit log for the guild.
+	 *
+	 * @return The full audit log for the guild.
+	 */
+	AuditLog getAuditLog();
+
+	/**
+	 * Gets the audit log with entries for the given action type.
+	 *
+	 * @param actionType The action type of the desired entries.
+	 * @return The audit log with entries for the given action type.
+	 */
+	AuditLog getAuditLog(ActionType actionType);
+
+	/**
+	 * Gets the audit log with entries with the given responsible user.
+	 *
+	 * @param user The responsible user of the desired entries.
+	 * @return The audit log with entries with the given responsible user.
+	 */
+	AuditLog getAuditLog(IUser user);
```

Comment:
If you are gonna keep the `getEntriesByResponsibleUser` stuff this should have consistent naming